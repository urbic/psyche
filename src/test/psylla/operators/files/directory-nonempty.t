'src/test/data/DIRECTORY-NONEMPTY' files { cmp } sorted { ? } forall
