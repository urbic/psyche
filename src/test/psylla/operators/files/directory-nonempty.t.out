'src/test/data/DIRECTORY-NONEMPTY/DIR-1'
'src/test/data/DIRECTORY-NONEMPTY/DIR-2'
'src/test/data/DIRECTORY-NONEMPTY/FILE-1'
'src/test/data/DIRECTORY-NONEMPTY/FILE-2'
